src/loaders/ObjectLoader.js

```diff
@@ -26,11 +28,17 @@ import { MaterialLoader } from './MaterialLoader';
 import { BufferGeometryLoader } from './BufferGeometryLoader';
 import { JSONLoader } from './JSONLoader';
 import { XHRLoader } from './XHRLoader';
+import * as CoreGeometry from '../core/Geometries';
+import * as ExtraGeometry from '../extras/geometries/Geometries';
 
 /**
  * @author mrdoob / http://mrdoob.com/
  */
 
+function resolveGeometryType ( type ) {
```

Comment:
Is this function actually necessary? As far as I can see the function is only used for geometry types which are part of the `ExtraGeometry` collection.